src/main/java/de/retest/ui/image/ImageUtils.java

```diff
@@ -266,12 +268,12 @@ public static String getTextFromIcon( final Icon icon ) {
 			// Normalize file:/-URLs to their base name
 			if ( result.startsWith( "jar:" ) ) {
 				result = result.replace( "jar:", "" );
```

Comment:
Then we should extract other URI schemes (e.g. `jar:`) as well; with meaningful names: `FILE_URI_SCHEME`, `JAR_URI_SCHEME`, …